src/components/App.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] You shouldn't disable linters. Please delete it and fix the error.
For stateless function,you can check this [link](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/prefer-stateless-function.md)

Reply:
Refactoring to functional components is a separate milestone of this project.